### ISSUE TITLE
Cast export support

### ISSUE THREAD

**Issue by dtzxporter** [dtzxporter/cast#31]

**Comment by dtzxporter** [comment]
Blender supports exporting animations.

**Comment by dtzxporter** [comment]
#30 related

**Comment by SushiiZ** [comment]
hey! just an update, been using the maya exporter portion of this for a bit and it works great, but there are a few caveats that would be cool to have support for, first off, when exporting from maya and importing in blender, it doesn't fix up Y=Up to Z=Up, causing bones that face upwards to be on their side, this only seems to affect the root bone and exporting only with all children of the root bone works fine, and then the other thing is it doesn't import with the scenes fps, it just imports at 1fps, which obviously isn't ideal, aside from that, it works great!

**Comment by dtzxporter** [comment]
I will fix the FPS export, though, I don't know if I want to include any orientation changes in cast, I think leaving it as-is will be less error prone...

**Comment by SushiiZ** [comment]
Yeah I'm not sure how .seanim handled it, it's not a big deal tbh but just figured I'd mention it

**Comment by dtzxporter** [comment]
Todo: Needs support for exporting blendshapes.
